### PR TITLE
enabled allprops flag; for donor insertions, limit inference of rando…

### PR DESCRIPTION
**Pull request by McMenemy**
…m insertions to +4bp

### CODE REVIEW

File: ice/analysis.py [hunk 1]
```diff
@@ -74,9 +74,10 @@ def single_sanger_analysis(control_path, sample_path, base_outputname, guide, do
     sa.initialize_with(control_path=control_path,
                        edited_path=sample_path,
                        gRNA_sequences=guide,
-                       indel_max_size=15,
+                       indel_max_size=20,
```
Comment (McMenemy) [inline ice/analysis.py]: What was the rational for changing this to 20?

File: ice/classes/sanger_object.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def initialize_from_path(self, ab1_file_path, *args, **kwargs):
         if np.count_nonzero(phred_scores) == 0 or override_quality_score:
             self.phred_scores = self.estimate_quality_scores()
 
-    def find_alignable_window(self, window_size=30, QUAL_CUTOFF=50):
+    def find_alignable_window(self, window_size=30, QUAL_CUTOFF=40):
```
Comment (McMenemy) [inline ice/classes/sanger_object.py]: Reasoning for lowering QUAL_CUTOFF?

File: ice/analysis.py (hunk 1 above)
Comment (hsiaut) [reply ice/analysis.py]: We had customer samples that were returning incorrect results (their confirmed deletion size was outside the bounds of edit proposal range).  We should probably increase indel_max_size to 30, but I did a more conservative increase to 20 for now.

Comment (hsiaut) [reply ice/analysis.py]: COM-1115

File: ice/classes/sanger_analysis.py [hunk 3]
```diff
@@ -669,7 +678,7 @@ def analyze_and_rank(self):
 
         # create totals:
 
-        aggregated_indel = {}
+        aggregated_indel = defaultdict(float)
```
Comment (McMenemy) [inline ice/classes/sanger_analysis.py]: Should be able to delete the if statement on line 685 since you changed to defaultdict

**Pull request merged**